app/Http/Controllers/ReportGroupController.php

```
@@ -40,12 +39,23 @@ public function searchReportGroups(ReportGroupSearchRequest $request)
     public function store(ReportGroupStoreRequest $request)
     {
         $validatedData = $request->validated();
-
-        return new JsonResponse([
-            'success' => true,
-            'message' => 'Report Group created successfully',
-            'data' => ReportGroup::create($validatedData),
-        ], 201);
+        try {
+            DB::beginTransaction();
```

PHPMD (2.15.0)

Avoid using static access to class '\DB' in method 'store'. (undefined)

(StaticAccess)